tests/datasetsPagetesting.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { expect } from "@playwright/test";
+import { Browser, BrowserContext, chromium } from 'playwright';
+import { test } from "../fixture/fixture";
+import { LoginPage } from "../page-object/pages/LoginPage";
+import { DatasetsPage } from "../page-object/pages/DatasetsPage";
+import { deleteFile } from "../helpers/helpers";
+import fs from 'fs';
+
+
+test('Notebook opening', async ({ page }) => {
+  const loginPage = new LoginPage(page);
+  await loginPage.login();
+  const datasetsPage = new DatasetsPage(page);
+  await page.waitForLoadState();
+  await datasetsPage.openDatasetsPage();
+  const searchInputLocator = page.locator('Search datasets');
+  await page.getByPlaceholder('Search datasets').click();
+  await page.getByPlaceholder('Search datasets').fill('grand-theft-auto-v');
+  await page.locator("(//div[@class='sc-jbicFF hIvzur'])[1]").click();
+  await datasetsPage.newNotebookButton.click();
+
+  const element = await page.locator('//div[contains(@class, "sc-dBnJEZ")]');
+  await page.waitForSelector('//div[contains(@class, "sc-dBnJEZ")]');
```

Comment:
Here we need some solution for new opened tab's content verification. Test fails because awaited selector is on the new opened tab